infrastructure/nomad/playbooks/templates/jobs/contracts-deployer.nomad.j2

```diff
@@ -92,6 +97,8 @@ job "{{ job.name }}" {
 
       template {
         data = <<-EOH
+          L1_FINALIZATION_FEE="1"
```

Comment:
These values are all good for devnet, but for holesky and mainnet deployment these values will need to be higher (on average, over time) than what the relayer pays to call `finalizeTransfer`

Reply:
What amount would be sufficient for the mainnet?

Reply:
We'll need to test or calculate what the relayer pays in ETH to call finalizeTransfer on average, over time. Lets call that `avgCost` for each chain. FinalizationFee must be set to 2*`avgCost` or some margin of safety above the cost that the relayer will incur on average for each tx